### ISSUE TITLE
Error identification (WCAG 3.3.1)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1242]
**Request:** The intent of this Success Criterion is to ensure that users are aware that an error has occurred and can determine what is wrong. The error message should be as specific as possible. In the case of an unsuccessful form submission, re-displaying the form and indicating the fields in error is insufficient for some users to perceive that an error has occurred. Screen reader users, for example, will not know there was an error until they encounter one of the indicators. If an input error is automatically detected, the item that is in error is identified and the error is described to the user in text.

**Issues:** 

1. https://slidewiki.org/ 
- In the "Login" dialog, the error message is displayed (if the e-mail address or password is not entered correctly) below the form fields. --> **This also effects other pages!**
--> The error message should be at the top of the page. In addition, the relevant form field is clearly highlighted. Alternatively, error messages that are positioned close to the form field, but are not part of the label, can be linked with `aria-labeledby` or `aria-describedby`.

2. https://slidewiki.org/contactus
- If none of the required form fields are filled in, an error message only appears in the field "Email". This is a pop-up that does not remain on the relevant field when scrolling.
- If you leave out the "Type of report" field (all other necessary fields are otherwise filled out), no error message appears. The contact form cannot be sent.
--> The error messages should be designed consistently within and across pages. The implementation on the page "Add Deck" could serve as a model.

**Comment by bwulff** [comment]
Dupe of #1202
